.gitignore

```diff
@@ -103,3 +103,4 @@ dump.rdb
 
 # pry configuration file
 .pryrc
+*.swp
```

Comment:
nitpick, non-blocking: I'm not opposed to adding this ignore pattern (it looks like a vim local save file), but it's not related to the PR (it probably makes sense on your global/user gitignore than any of the projects you're working on).

Reply:
Agreed, any chance we could remove this from here @williamkennedy?